modules/ROOT/pages/siebel/siebel-connector.adoc

```diff
@@ -154,15 +154,48 @@ Displays all records that includes a valid owner. For example, it includes the r
 [NOTE]
 Limit the number of objects to retrieve through DataSense to a few objects using search specifications, otherwise retrieving metadata slows down Studio.
 
-== Log Requests and Responses [[log-requests-responses]]
+[[log-requests-responses]]
+== Log Requests and Responses 
 
-To log requests and responses when using the connector, configure the `log4j.xml` configuration file for the Mule app as follows:
+To log requests and responses when using the connector, add the following line to the `Loggers` element of the `log4j2.xml` configuration file for the Mule app:
 
 [source,xml,linenums]
 ----
 <AsyncLogger name="org.mule.modules.siebel.internal.service"
    level="DEBUG"/>
 ----
+
+For example:
```

Comment:
GLOBALLY avoid this brief fragment to introduce an example.
Either end the introductory text with a colon or make this fragment more descriptive, telling what this is an example of.